src/styles/variables.ts

```diff
@@ -203,7 +203,7 @@ export default {
     sectionIllustrationHeight: 220,
     photoUploadPopoverWidth: 335,
     onboardingModalWidth: 500,
-    welcomeVideoDelay: 500,
+    welcomeVideoDelay: 1000,
```

Comment:
It does not feel very reliable way here but I guess its ok for now. If the device is slower I wonder if the crash couls still occur

Reply:
Hmm... I tried it on some Android devices. It works fine even with 500. So we can try to fix it better later.

Reply:
Yeah definitely nothing to worry about now, let see if we will notice this issue in future